prettyprinters/human/human.go

```diff
@@ -70,14 +70,15 @@ func (p *Printer) StartPP(g *graph.Graph) (pp.Renderable, error) {
 
 // FinishPP provides summary statistics about the printed graph
 func (p *Printer) FinishPP(g *graph.Graph) (pp.Renderable, error) {
-	tmpl, err := p.template("{{if gt (len .Errors) 0}}{{red \"Summary\"}}{{else}}{{green \"Summary\"}}{{end}}: {{len .Errors}} errors, {{.ChangesCount}} changes{{if .Errors}}\n{{range .Errors}}\n * {{.}}{{end}}{{end}}\n")
+	tmpl, err := p.template("{{if .Errors}}Errors:\n{{range .Errors}} * {{.}}\n{{end}}\n{{end}}{{if .DependencyErrors}}Failed due to failing dependency:\n{{range .DependencyErrors}} * {{.}}\n{{end}}\n{{end}}{{if gt (len .Errors) 0}}{{red \"Summary\"}}{{else}}{{green \"Summary\"}}{{end}}: {{len .Errors}} errors, {{.ChangesCount}} changes{{if .DependencyErrors}}, {{len .DependencyErrors}} dependency errors{{end}}\n")
```

Comment:
It might be more readable if this were broken up into multiple lines as long as that doesn't break anything.